### DISCUSSION TITLE
List of random Quotes

### DISCUSSION THREAD

**Discussion by Anish-Chanda** [lukePeavey/quotable#121] [Ideas]
I went through the documentation but I couldn't find a way to get a list of random quotes, so for now I created a for loop which gets 5 random quotes and adds it to a list but I know that isn't a great idea, It would be great if we can retrieve a list of random quotes at once :)

**Comment by lukePeavey** [comment]
The API doesn't currently have a method to get a list of random quotes.

I'm think this would be a cool feature to add.
